### ISSUE TITLE
taxonomicName ChecklistBank ID

### ISSUE THREAD

**Issue by myrmoteras** [plazi/treatmentBank#96]
@gsautter might it be possible to add to add checklistBank ID for a treatment or `taxonomic Name ChecklistBank ID` to the Plazi stats? In a sense this is trivial, but it would add another bidirectional link to our collection. In BiCIKL it is figures that count.... kind of bidirectional links, number of bidirectional links

**Comment by gsautter** [comment]
We kind of already have that in https://tb.plazi.org/GgServer/srsStats ... "CoL Taxon Name ID" in "Taxonomic Data", as well as "Article ChecklistBank Dataset ID" in "Treatment & User Data" ... what else do you mean, or where should it be added?

**Comment by myrmoteras** [comment]
but we don't have the checklistBank taxon Name ID, which has a one to multiple relationship to CoL taxon Name ID.

And even though the id is trivial (treatmentUUID.taxon) it is relevenat, because the relationship to to CoL Taxon Name ID is inferred by us, but not confirmed by CoL, it is one more bidirectional link we have in our repertoire.

**Comment by gsautter** [comment]
Not sure here, really ... the "front end" of CLB is CoL, for what I understand ...

**Comment by myrmoteras** [comment]
no, CLB is the staging area to get checklists incorporated into COL. The name matches have to be accepted currently by a curator, and cant' be done by machine yet, this is why we are quickly building up a huge backlog.

At some point, then the link between a CLB-identifier and the COL-ID will be done, and in fact should replace our inferred use of COL ID.

**Comment by gsautter** [comment]
> At some point, then the link between a CLB-identifier and the COL-ID will be done, and in fact should replace our inferred use of COL ID.

That should be good, soon as it comes online ... might need to use a delayed lookup strategy akin to what we use for getting back the individual record keys from GBIF after exporting a DwCA.

**Comment by gsautter** [comment]
> no, CLB is the staging area to get checklists incorporated into COL. The name matches have to be accepted currently by a curator, and cant' be done by machine yet, this is why we are quickly building up a huge backlog.

Fair enough ... just a question whether or not it's sensible to link into this "staging area", as, like the term suggests, it's an intermediate stage rather than a final output ...

**Comment by myrmoteras** [comment]
It is important to link there, because the CLB ID Wii stick, because this name usage is the "sec." We even use in BLR and which is unique.

Also the ID will not change because they use our UUID.taxon as ID

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by gsautter** [comment]
OK, fair enough ... might take a few days to introduce this, though, need to add some new mechanism to the stats engine to support such things while avoiding redundant data.
